acceptance/suites/pre_suite/compat_common/30_install_dev_repos.rb

```diff
@@ -1,4 +1,4 @@
-install_opts = options.merge({ :dev_builds_repos => ["PC1"]})
+install_opts = options.merge({ :dev_builds_repos => ["puppet"]})
```

Comment:
FWIW, when I ran the Puppet Server smoke tests against this PR, the tests all passed for me.  I don't know with this line and the related ones below it do exactly with beaker but the changes don't appear to adversely affect the job, in any event.  Maybe this doesn't typically matter in the Puppet Server pipeline anyway since we have it download internal repo configs directly from the builds.puppetlabs server?